shelley/chain-and-ledger/formal-spec/notation.tex

```diff
@@ -40,9 +40,21 @@ \section{Notation}
   \item[Option type] An option type in type $A$ is denoted as $A^? = A + \Nothing$. The
     $A$ case corresponds to the case when there is a value of type $A$ and the $1$
     case corresponds to the case when there is no value.
-
 \end{description}
 
+\begin{note}
```

Comment:
maybe we can just list the let binding notation in the same itemized list with the other notation instead of making a note?